docs/examples/Parameters/Parameters.ipynb

```diff
@@ -23,55 +50,46 @@
     "import numpy as np\n",
     "from typing import Optional\n",
     "\n",
-    "from qcodes.instrument.base import InstrumentBase\n",
-    "from qcodes.instrument.parameter import Parameter, ArrayParameter, MultiParameter, ManualParameter\n",
-    "from qcodes.tests.instrument_mocks import DummyInstrument\n",
-    "from qcodes.utils import validators"
+    "from qcodes.instrument.base import (\n",
+    "    InstrumentBase\n",
+    ")\n",
+    "from qcodes.instrument.parameter import (\n",
+    "    Parameter, \n",
+    "    ArrayParameter,\n",
+    "    MultiParameter, \n",
+    "    ManualParameter\n",
+    ")\n",
+    "from qcodes.tests.instrument_mocks import (\n",
+    "    DummyInstrument,\n",
+    ")\n",
+    "from qcodes.utils import (\n",
+    "    validators\n",
+    ")"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "QCoDeS provides the following classes of built-in parameters:\n",
+    "## Simple parameter subclass\n",
```

Comment:
should these be on the same header level with the ``## Examples`` or inside it? it's a bit confusing now. e.g. i'd expect the instrument parameters section at the bottom to be on the same level as "simple parameter subclass" and "virtual parameters"